src/rational/poly_over_q/properties.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright © 2023 Phil Milewski
+//
+// This file is part of qFALL-math.
+//
+// qFALL-math is free software: you can redistribute it and/or modify it under
+// the terms of the Mozilla Public License Version 2.0 as published by the
+// Mozilla Foundation. See <https://mozilla.org/en-US/MPL/2.0/>.
+
+//! This module includes functionality about properties of [`PolyOverQ`] instances.
+
+use flint_sys::fmpq_poly::{fmpq_poly_degree, fmpq_poly_is_one};
+
+use super::PolyOverQ;
+
+impl PolyOverQ {
+    /// Checks if a [`PolyOverQ`] is the constant polynomial with coefficient `1`.
+    ///
+    /// Returns true if the first coefficient is `1` and is the only coefficient
```

Comment:
```suggestion
    /// Returns true if the first coefficient is `1` and is the only coefficient.
```
Also for the other functions